### PR TITLE
modify co2store gaswater case

### PR DESCRIPTION
**Pull request by totto82**
The new case includes vaporized water and EQUIL initialization.

### CODE REVIEW

File: co2store/CO2STORE_GW.DATA [hunk 1]
```diff
@@ -5,13 +5,13 @@
 
 -- Copyright (C) 2021 NORCE
 
--- Test case for gas + water + disgasw (dissolved gas in water phase)
+-- Test case for gas + water + disgasw + vapwat (dissolved gas in water phase and vapporized water in gas phase)
```
Comment (akva2) [inline co2store/CO2STORE_GW.DATA:8]: vaporized

**Comment by totto82** [comment]
jenkins build this update_data please

**Comment by totto82** [comment]
jenkins build this please

**Comment by bska** [comment]
> update data from this PR [in opm-tests],

Yeah, for whatever reason that does not appear to do the right thing.  The resulting PR (#1049) did not have any updated reference solutions.

**Comment by akva2** [comment]
yeah, i guess it's because i started implementing pr from pr but it is somehow broken and i haven't looked  at it again.

**Comment by totto82** [comment]
As you probably noticed I just updated the reference solution locally and pushed it as in the old days. Can we merge this now? I plan to use this for demonstration tomorrow and just wanted to clean in up first.

**Pull request merged**